sympy/stats/crv_types.py

```diff
@@ -1020,6 +1020,11 @@ class FDistributionDistribution(SingleContinuousDistribution):
 
     set = Interval(0, oo)
 
+    @staticmethod
+    def check(d1, d2):
+        _value_check(d1 > 0 and d1.is_integer, "Degrees of freedom d1 must be positive integer.")
```

Comment:
The line could be split with the string on the next line.

Reply:
Ok, I'll do it